# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/base/test/jsmodules/import_404.mjs
# Commit: e47abb2b8ac7
# Full Hash: e47abb2b8ac70efe89371c67766def82374c0f36
# Author: Alexandru Marc <amarc@mozilla.com>
# Date: 2025-09-27 12:29 +0300
# Description:
#   Revert "Bug 1968890 - Part 9: Add test for dynamic import. r=jonco" as potential cause of javascript engine crashes (bug 1991268)
# ==============================================================================

diff -r 7e2611c97326 -r e47abb2b8ac7 dom/base/test/jsmodules/import_404.mjs
--- a/dom/base/test/jsmodules/import_404.mjs	Sat Sep 27 11:41:54 2025 +0300
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,2 +0,0 @@
-/* eslint-disable import/no-unresolved */
-import { A } from "./404.js";